themes/atomic/atomic.theme.bash

```diff
@@ -183,7 +224,27 @@ ___atomic_prompt_battery() {
 	info+=$batp
 	[ "$info" == "+100" ] && info="AC"
 	printf "%s|%s|%s|%s" "${color}" "${info}" "${bold_white}" "${box}"
-}
+	} ;; 
+  linux*) ___atomic_prompt_battery() {
+	[ ! -e "$BASH_IT"/plugins/enabled/battery.plugin.bash ] ||
```

Comment:
Again, it looks like this code block is pretty much similar to the `darwin` case. This could probably be moved into a single case.